### DISCUSSION TITLE
GoG offline mode?

### DISCUSSION THREAD

**Discussion by justind3d** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#43] [Q&A]
So the only way I can get this all working is by turning off my internet currently. I am running NMS through GoG, I have multiplayer turned off and am launching through the folder, not Galaxy. However, when I launch, NMS immediately overwrites the JSON file and the cursed expedition loads. I tried to disable my internet (just to test, it's not feasible to turn it off for the duration of the gameplay) and the JSON works and I can load the older expeditions, However, as soon (like within seconds) as I turn on the internet, the expedition ends and I get booted out saying it's over. Obviously even with NMS launching outside of galaxy and with multiplayer turned off, it is still in constant communication with the server. My question is do I have to actually disable my internet for this workaround to function or am I missing something that will disable that server communication? Thank you!

**Comment by cwmonkey** [comment]
@justind3d

~~Firstly, you don't need to disable your internet for the duration of your time in NMS, just for the time when you open it.~~

Secondly, you could theoretically block the exe via some sort of firewall setting. For example in the Windows firewall you can just open the "Allowed apps" and disallow No Man's Sky from being able to communicate to public networks. Whatever operating system you have, it should be easy enough to block specific apps and perhaps make a toggle script you can run to do it for you, just google around.

Also worth noting I'm not familiar with GoG, so there could be other alternatives getting "offline mode" working out there somewhere.

**Reply by milchmann88** [reply]
Just a heads up: It seems like you now need to be offline permanently for this to work. I just got kicked from my expedition when I put steam into online mode again.  And it didn't create any restore point / autosave to go back.

**Reply by cwmonkey** [reply]
@milchmann88 Oh damn, confirmed! Well that sucks.

**Reply by cwmonkey** [reply]
Thanks for that info, @milchmann88. I added some verbiage to https://cwmonkey.github.io/nms-expeditions/ and I'll ask @leodium make appropriate updates elsewhere.
